compiler/AST/symbol.cpp

```diff
@@ -1598,6 +1598,16 @@ int FnSymbol::hasGenericFormals() const {
   bool hasGenericDefaults =  true;
   int  retval             =     0;
 
+  bool inInitRes = false;
```

Comment:
"res" is a shortening of resolution because I didn't want to make the argument name that long